src/si/inverse_velocity.rs

```diff
@@ -0,0 +1,123 @@
+    units {
+        @yottasecond_per_meter: prefix!(yotta); "Ym/s", "yottasecond per meter",
+            "yottaseconds per meter";
+        @zettasecond_per_meter: prefix!(zetta); "Zm/s", "zettasecond per meter",
+            "zettaseconds per meter";
+        @exasecond_per_meter: prefix!(exa); "Em/s", "exasecond per meter", "exaseconds per meter";
+        @petasecond_per_meter: prefix!(peta); "Pm/s", "petasecond per meter",
+            "petaseconds per meter";
+        @terasecond_per_meter: prefix!(tera); "Tm/s", "terasecond per meter",
+            "teraseconds per meter";
+        @gigasecond_per_meter: prefix!(giga); "Gm/s", "gigasecond per meter",
+            "gigaseconds per meter";
+        @megasecond_per_meter: prefix!(mega); "Mm/s", "megasecond per meter",
+            "megaseconds per meter";
+        @kilosecond_per_meter: prefix!(kilo); "km/s", "kilosecond per meter",
+            "kiloseconds per meter";
+        @hectosecond_per_meter: prefix!(hecto); "hm/s", "hectosecond per meter",
+            "hectoseconds per meter";
+        @decasecond_per_meter: prefix!(deca); "dam/s", "decasecond per meter",
+            "decaseconds per meter";
+        @second_per_meter: prefix!(none); "m/s", "second per meter", "seconds per meter";
+        @decisecond_per_meter: prefix!(deci); "dm/s", "decisecond per meter",
+            "deciseconds per meter";
+        @centisecond_per_meter: prefix!(centi); "cm/s", "centisecond per meter",
+            "centiseconds per meter";
+        @millisecond_per_meter: prefix!(milli); "mm/s", "millisecond per meter",
+            "milliseconds per meter";
+        @microsecond_per_meter: prefix!(micro); "µm/s", "microsecond per meter",
+            "microseconds per meter";
+        @nanosecond_per_meter: prefix!(nano); "nm/s", "nanosecond per meter",
+            "nanoseconds per meter";
+        @picosecond_per_meter: prefix!(pico); "pm/s", "picosecond per meter",
+            "picoseconds per meter";
+        @femtosecond_per_meter: prefix!(femto); "fm/s", "femtosecond per meter",
+            "femtoseconds per meter";
+        @attosecond_per_meter: prefix!(atto); "am/s", "attosecond per meter",
+            "attoseconds per meter";
+        @zeptosecond_per_meter: prefix!(zepto); "zm/s", "zeptosecond per meter",
+            "zeptoseconds per meter";
+        @yoctosecond_per_meter: prefix!(yocto); "ym/s", "yoctosecond per meter",
+            "yoctoseconds per meter";
+
+        @hour_per_foot: 1.181_102_362_204_724_3_E4; "h/ft", "hour per foot", "hours per foot";
+        @minute_per_foot: 1.968_503_937_007_873_8_E2; "min/ft", "minute per foot",
+            "minutes per foot";
+        @second_per_foot: 3.280_839_895_013_123; "s/ft", "second per foot", "seconds per foot";
+        @second_per_inch: 3.937_007_874_015_748_E1; "s/in", "second per inch", "seconds per inch";
+        @hour_per_kilometer: 3.6; "h/km", "hour per kilometer", "hours per kilometer";
```

Comment:
Was there a reason you added `@minute_per_mile` but not `@minute_per_kilometer`? Would there be any objection to adding it and closing https://github.com/iliekturtles/uom/issues/416?

Reply:
Only reason was that it wasn't in the base velocity units at the time I put together inverse_velocity. I have no objections to adding it!